packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -107,6 +100,25 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
       flattened: Option[];
     }>(groupedOptions);
 
+    const handleClose = (selection?: Option | null) => {
+      if (!selection) {
+        if (inputValue !== selectedOption?.title) {
+          setInputValue(selectedOption?.title || "");
+          setResults(groupedOptions);
+          setActiveIdx(0);
+        }
+      }
+
+      if (onClose && isOpened) onClose(selection || selectedOption);
+      setIsOpened(false);
+    };
+
+    const handleOutsideClick = () => {
```

Comment:
Sometimes we need to call `handleClose` with a parameter of type `Option | null`. If we pass that function to `useClickOutside`, it will be type incompatible, because `useClickOutside` expects a function that receives an event. With this, and the reason mentioned below, I think this should be the reasonable approach. WDYT?

Reply:
wdym by type incompatiable? adding it like that should be ok, no? :) 

*useClickOutside(labelRef, () => handleClose());*

Reply:
Yes, but then on all the onClick I would have to leave the same, WDYT? We win in one side to lose on the other 😅

Reply:
not sure if I understand wdym by leave the same. This function is used only once inside useClickOutside, no need for that, simply call it as I showed :)

Reply:
Yes, sorry, I wasn't too clear. Besides `useOutsideClick` On the Modal we have an `onClose` prop, and we also have a Button for closing and a ModalCloseButton, both with `onClick`. They all need to do the same as `useOutsideClick`: call `handleClose` with no args.

As it is right now, I am defining `handleCloseClick` once that does exactly that, and I use that definition on all 4 scenarios. But if I do as you suggest, I will have to define `() => handleClose()` four different times, unnecessarily, because I could just reuse it.